### PR TITLE
Fix repeated config generation overloading /queue

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: tendrl/performance_monitoring/sds/ceph/ceph_plugin.py [hunk 1]
```diff
@@ -117,8 +119,9 @@ def configure_monitoring(self, sds_tendrl_context):
                 ):
                     node_plugins = self.configured_nodes.get(node_id, [])
                     node_plugins.append(
-                        "ceph_node_network_throughput"
+                        "tendrl_ceph_node_network_throughput"
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: checked plugin name correctness at other places in the code?

**Comment by r0h4n** [comment]
@anmolbabu file an actual issue, link it to this PR by tendrl-bug-id: Tendrl/performance-monitoring#issue_id

File: tendrl/performance_monitoring/sds/ceph/ceph_plugin.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: Yes. This just remained unnoticed ..

File: tendrl/performance_monitoring/sds/ceph/ceph_plugin.py [hunk 2]
```diff
@@ -80,6 +81,7 @@ def configure_monitoring(self, sds_tendrl_context):
                     node_plugins.append(
                         "ceph_cluster_iops"
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: is this plugin name correct? or is there supposed to be a "tendrl_" at start

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/ceph/ceph_plugin.py]: Aah now fixed everywhere

**Comment by anmolbabu** [comment]
Verified

**Pull request merged**